frontend/scss/_common.scss

```diff
@@ -57,7 +57,10 @@ img {
 }
 
 .blue-section {
-  background-color: $hover-blue;
+  background-image: url(../images/news-lab.png);
```

Comment:
Please use a different class for this, the properties you added here will affect other pages (e.g the homepage)

Reply:
Ok.done